webapp/package.json

```diff
@@ -6,7 +6,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "develop": "NEXT_PUBLIC_HOST=\"http://127.0.0.1:36462\" next dev",
     "mock": "NEXT_PUBLIC_HOST='' next dev",
-    "build": "next build",
+    "build": "NEXT_PUBLIC_HOST='' next build && next export -o build/web/",
```

Comment:
Sometimes, I wanted to run `next` commands directly on the command line.
If `NEXT_PUBLIC_HOST=''` isn't set, `undefined` is prefixed to Ajax paths.
Should we use if-else condition or Elvis operator to avoid such cases?